simulators/history/rpc-compat/src/main.rs

```diff
@@ -469,10 +600,16 @@ dyn_async! {
 
 dyn_async! {
     // test that a node will return a AbsentContent via RecursiveFindContent when the data doesn't exist
-    async fn test_recursive_find_content_content_absent<'a> (client: Client) {
+    async fn test_recursive_find_content_content_absent<'a>(clients: Vec<Client>, _: Option<Vec<(String, String)>>) {
+        let client = match clients.into_iter().next() {
```

Comment:
Maybe writing a small utility function could be useful here to cut down on some of the repeated code?

eg...
```rust
fn get_expected_client(clients: Vec<Client>) -> Client {
    match clients.into_iter().next() {
       Some((client)) => client,
       None => {
           panic!("Unable.... ");
       }
    }
}
```

Or maybe there's a cleaner solution I'm missing, just seems to me like there's an opportunity here to get rid of some of the repeated code.